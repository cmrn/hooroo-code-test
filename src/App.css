.App {
  max-width: 840px;
  margin: 0 auto;
  padding: 20px;
}

.App-logo {
  height: 40px;
}
