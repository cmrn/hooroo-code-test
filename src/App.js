import React, { Component } from 'react';
import logo from './qantas-logo.png';
import styles from './App.css';

class App extends React.Component {
  render() {
    return (
      <div className="App">
        <header>
          <img src={logo} className="App-logo" alt="Qantas" />
        </header>
      </div>
    );
  }
}

export default App;
