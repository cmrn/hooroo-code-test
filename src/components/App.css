.App {
  min-width: 700px;
  max-width: 900px;
  margin: 0 auto;
  padding: 20px;
}

.App-logo {
  height: 40px;
  margin-left: -15px;
}
