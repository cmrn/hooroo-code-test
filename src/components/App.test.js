import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import App from './App';

it('renders the search page and header', () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(<App />);
  expect(tree).toMatchSnapshot();
});