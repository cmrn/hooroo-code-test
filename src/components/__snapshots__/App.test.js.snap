// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the search page and header 1`] = `
<div
  className="App"
>
  <header>
    <img
      alt="Qantas"
      className="App-logo"
      src="qantas-logo.png"
    />
  </header>
  <SearchContainer />
</div>
`;
